plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -665,13 +665,13 @@ protected JdbcOutputTableHandle createTable(ConnectorSession session, ConnectorT
         }
     }
 
-    protected String createTableSql(RemoteTableName remoteTableName, List<String> columns, ConnectorTableMetadata tableMetadata)
```

Comment:
Can we deprecate the old one and by default we could call the old one in the new one?

```
  protected List<String> createTableSqls(RemoteTableName remoteTableName, List<String> columns, ConnectorTableMetadata tableMetadata)
  {
      return ImmutableList.of(createTableSql(remoteTableName, columnList.build(), tableMetadata));
}      
```
?

This `trino-base-jdbc` framework is kind of library where people are using this outside of Trino repo, so such change could break them.

Reply:
Make sense 👍